turtlesim/src/turtle.cpp

```diff
@@ -34,13 +34,20 @@
 #include <QColor>
 #include <QRgb>
 
+#define TWO_PI 6.283185307179586
```

Comment:
Why not use the definition of `PI` from `turtlesim/turtle.h`?

Reply:
See 5f614f0